src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsnonassociationsapi/resource/NonAssociationsResource.kt

```diff
@@ -83,12 +85,32 @@ class NonAssociationsResource(
     )
     @RequestParam(required = false)
     includeOtherPrisons: Boolean,
+
+    @Schema(
+      description = "Sort non-associations by",
+      required = false,
+      defaultValue = "WHEN_CREATED",
+      example = "LAST_NAME",
+      allowableValues = [
+        "WHEN_CREATED",
+        "LAST_NAME",
+        "FIRST_NAME",
```

Comment:
i doubt we'd ever use first name, but i'm totes fine having it as an option. is it worth having prisoner number tho?

Reply:
no idea, but we could add easily if we needed to.

Reply:
[Ability to sort by prisoner number](https://github.com/ministryofjustice/hmpps-non-associations-api/pull/37/commits/c73fd951eed644ed80afc9843a8ec915457c3f34)